biomage/account/account.py

```diff
@@ -137,19 +140,60 @@ def _create_user(full_name, email, password, userpool, overwrite=False):
     email = email.lower()
 
     error = create_account(full_name, email, userpool)
-    # if the user already exists, just proceed and change the password
-    # this way, when there's an error creating a list you can just
-    # re-run the whole script and get the correct tmp passwords
+    # Return error if we are not overwriting user password
     if error and ("usernameExistsException" not in str(error) or not overwrite):
         return error
 
-    if error:
```

Comment:
this shouldn't be removed, it should become a `elif error:` so that we catch possible errors (that are not user already exists errors)

Reply:
I remove this because otherwise the overwrite flag is useless. Consider a case where we want to overwrite the users's password (overwrite = true), and then find a user that exists in the database, i.e.

overwrite = True
error = ...."usernameExistsException"....

1. On line 144, the overwrite flag will cause the if statement on line 144 to evaluate to false because  `"usernameExistsException" not in str(error) is false and not overwritten is false`. 
2. Execution proceeds to line 146. This line will evaluate to true because error is a string. Because of that the function returns, and the user password is not overwritten, which is not what we want.

Reply:
yes that's right, mb